_events/sr2017/bristol-tech-day-february.md

```diff
@@ -13,11 +13,11 @@ with other teams!
 
 ## Important Notice
 
-**If your team wants to attend, you need to bring your team leader or another
```

Comment:
Issue: At the time, we were calling them "leaders" - I think we should leave historical references as-is.

Reply:
https://github.com/srobo/website/pull/575/commits/34d40489dfb23d378c92b8c37eb590e35765b133